File: IsciProblemi.swift
//  Created by Kağan on 15.01.2017.
//  Copyright © 2017 Kağan Utku Kılıçlı. All rights reserved.

import Foundation


print("Bir isci bu isi kac gunde yapar?")
if var birIsciKacGundeYapar = Int(readLine(strippingNewline: true)!) {
    print("İsci sayisini giriniz")
    if var kacIscıCalisicak = Int(readLine(strippingNewline: true)!) {
        print("isin bitme süresi \(birIsciKacGundeYapar / kacIscıCalisicak) gundur.")
    }
}

File: IsciProblemi.c
#include <stdio.h>
#include <conio.h>

int main(){
	printf("bir isci yaptigi isi kac gunde bitirebilmektedir: ");
	int birisci;
	scanf("%d",&birisci);
	printf("toplam kac isci calisacak: ");
	int xisci;
	scanf("%d",&xisci);
	printf("isin bitme suresi %d gundur.",birisci/xisci);
	getch();
}
